easybuild/easyconfigs/t/texinfo/texinfo-5.2-GCC-4.8.2.eb

```diff
@@ -16,6 +11,14 @@ sources = [SOURCELOWER_TAR_XZ]
 
 toolchain = {'name': 'GCC', 'version': '4.8.2'}
 
+osdependencies = ['texlive']
+
+preinstallopts = "mkdir -p %(installdir)s/texmf/{doc,tex}/{context,eplain,generic,latex,lualatex,plain,xelatex,xetex} && "
+preinstallopts += "make TEXMF=%(installdir)s/texmf install-tex && "
+
+# This will overwrite a users TEXMFHOME so this module should only really be used as a build dependency
+modextravars = {'TEXMFHOME': '%(installdir)s/texmf'}
```

Comment:
so, maybe it shouldn't be set by the module then?

where texinfo is used as a build dep, you can redefine `$TEXMFHOME` during the build (via `(pre)buildopts`, for example)